ckanext/xloader/jobs.py

```diff
@@ -245,10 +245,21 @@ def tabulator_load():
     logger.info("'use_type_guessing' mode is: %s", use_type_guessing)
     try:
         if use_type_guessing:
-            tabulator_load()
+            try:
+                tabulator_load()
+            except rq_timeouts.JobTimeoutException as e:
```

Comment:
This isn't catching timeouts from the tabulator_load on line 266. Perhaps move this block to the outer catch, line 267?

Reply:
moved to outer try/catch alongside the `FileCouldNotBeLoadedError` one.